packages/opentelemetry-resources/src/index.ts

```diff
@@ -14,22 +14,19 @@
  * limitations under the License.
  */
 
-export { Resource } from './Resource';
-export { IResource } from './IResource';
-export { defaultServiceName } from './platform';
-export { DetectorSync, Detector } from './types';
 export { ResourceDetectionConfig } from './config';
+export { detectResources } from './detect-resources';
 export {
   browserDetector,
   browserDetectorSync,
   envDetector,
-  envDetectorSync,
   hostDetector,
-  hostDetectorSync,
+  noopDetector,
```

Comment:
We did not have this exported, before - is this on purpose?

Reply:
I didn't notice it wasn't exported before. I don't think there's a big problem if we do export it, but it also doesn't seem to provide value so i'll remove it.